api/api-rules/violation_exceptions.list

```diff
@@ -1 +1,4 @@
 API rule violation: names_match,github.com/openkruise/kruise/pkg/apis/apps/v1alpha1,SidecarContainer,Container
+API rule violation: names_match,github.com/openkruise/kruise/pkg/apis/apps/v1alpha1,UnitedDeploymentCondition,LastTransitionTime
```

Comment:
You should resolve all these API rule violations as much as possible.

Reply:
done